ng/src/web/components/form/multiselect.js

```diff
@@ -162,7 +164,8 @@ class MultiSelect extends React.Component {
       className,
       disabled = false,
       items,
-      menuPosition,
+      menuPosition = 'adjust',
```

Comment:
Please adjust your commit message of f1f6e9ae5d982a50475d1bda2cd8fdc3a247b546 or split the commit into to separated commits. Your are not mentioning the setting a new default for menuPosition.